host/sgx/calls.c

```diff
@@ -898,6 +953,36 @@ oe_result_t oe_call_enclave_function(
         output_bytes_written);
 }
 
+/*
+**==============================================================================
+**
+** oe_switchless_call_enclave_function()
+**
+** Switchlessly call the enclave function specified by the given function-id in
+** the default function table.
+**
+**==============================================================================
+*/
+oe_result_t oe_switchless_call_enclave_function(
+    oe_enclave_t* enclave,
+    uint32_t function_id,
+    const void* input_buffer,
+    size_t input_buffer_size,
+    void* output_buffer,
+    size_t output_buffer_size,
+    size_t* output_bytes_written)
+{
+    return oe_switchless_call_enclave_function_by_table_id(
+        enclave,
+        OE_UINT64_MAX,
```

Comment:
For my understanding, Why is the table id OE_UINT64_MAX

Reply:
This is the convention used to identify the table generated by oeedger8r from an user EDL.

There are other internal tables (for sockets, file systems etc) that have predefined ids.

Reply:
`OE_UINT64_MAX` means we are searching all enclave function tables.